docs/source/topics/cache.rst

```diff
@@ -32,6 +32,25 @@ In addition, the cache is culled based on ``CULL_FREQUENCY`` when  ``add()``
 or ``set()`` is called, if ``MAX_ENTRIES`` is exceeded. See
 :ref:`django:cache_arguments` for an explanation of these two options.
 
+Cache entries include a HMAC signature to ensure data integrity by default.
+You can disable this by setting ``ENABLE_SIGNING`` to ``False``.  
+Signatures can also include an optional key and salt parameter by setting 
+``KEY`` and ``SALT`` repectively. Signatures are provided by the Blake2 hash
+function, making Key sizes limited to 64 bytes, and salt sizes limited to 16 
+bytes. If a key is not provided, cache entries will be signed using the 
+``SECRET_KEY``.
```

Comment:
I think it's important to be explicit about what  "data integrity" means. (i.e. If your database is compromised, it can lead to RCE.) 
- It should also be made clear that this behavior differs from Django's built-in database backends.
- It would be helpful to give some guidance about the performance implications.